### PR TITLE
Add the python path to environment variables

### PR DESCRIPTION
**Pull request by mrpau-julius**
I enhance the python-exe.bat, to set the python path and add to the environment variables.

### Reference 
#102

### CODE REVIEW

File: windows/python-setup/python-exe.bat [hunk 1]
```diff
@@ -3,6 +3,11 @@
 Set version=3.4.3
 Set python32Bit=python-%version%.msi
 Set python64Bit=python-%version%.amd64.msi
+Set pythonPath=%cd:~0,2%/Python34/Scripts
+
+rem Add the pythonPath to environment variables.
+:Add_python_Path
+Setx Path "%pythonPath%;%Path%"
```
Comment (benjaoming) [inline windows/python-setup/python-exe.bat]: Oh nice! That would be a huge step forwards.

* How many Windows systems does this work on?
* If Python is already installed (for instance if people are upgrading), will this be skipped?

Comment (mrpau-julius) [reply windows/python-setup/python-exe.bat]: > How many Windows systems does this work on?

It works on Windows 7, 8.1, 10 and Windows Server.

> If Python is already installed (for instance if people are upgrading), will this be skipped?

No, I will add a code to check if the python path exists in the environment variable.

**Comment by indirectlylit** [comment]
testing PR: https://github.com/learningequality/kolibri/pull/5705

One word of caution: this PR is coming from

[`mrpau:feature/102-make-kolibri.exe-runnable-in-cmd-prompt`](https://github.com/mrpau/kolibri-installer-windows/tree/feature/102-make-kolibri.exe-runnable-in-cmd-prompt)

but the Kolibri build above is coming from

[`learningequality:kolibri-path`](https://github.com/learningequality/kolibri-installer-windows/tree/kolibri-path)

Recommend making these the same so there is a single source of truth

**Comment by cpauya** [comment]
> Recommend making these the same so there is a single source of truth

Good catch @indirectlylit - thanks.

Will ask @mrpau-julius why he created another branch at https://github.com/learningequality/kolibri-installer-windows/tree/kolibri-path instead of just pointing the PR to the https://github.com/mrpau/kolibri-installer-windows/tree/feature/102-make-kolibri.exe-runnable-in-cmd-prompt branch.

**Comment by mrpau-julius** [comment]
I think @benjaoming create this https://github.com/learningequality/kolibri-installer-windows/tree/kolibri-path for testing purposes.

**Comment by indirectlylit** [comment]
closing - work will continue in feature branch under #107

**Pull request closed without merging**